composer.json

```diff
@@ -15,21 +15,21 @@
         }
     ],
     "require": {
-        "php": "^8.1.0",
+        "php": "^8.2.0",
```

Comment:
Please revert this also.

```suggestion
        "php": "^8.1",
```

Reply:
It won't work
<img width="791" alt="image" src="https://github.com/openai-php/symfony/assets/6862373/b38f93cc-77e5-485f-911b-ffc47dfce8f9">

Reply:
This is the minimum version of PHP to install this dependency. If you have a project with PHP 8.2, it satisfies the contraint on PHP 8.1.